pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -92,7 +94,12 @@ stdenv.mkDerivation rec {
   # yes, upstream really packages it as "vesktop" but uses "vencorddesktop" file names
   installPhase =
     let
-      libPath = lib.makeLibraryPath [ pipewire ];
+      # this is mainly required for venmic
+      libPath = lib.makeLibraryPath [
+        libpulseaudio
+        pipewire
+        gcc13Stdenv.cc.cc.lib
+      ];
```

Comment:
mostly looks good, but does venmic have natives for aarch64? maybe vendor venmic if need be?

Reply:
actually, aarch64 is already broken so it's not hurting much of anything. aarch64 being (possibly) broken could be addressed in a future pr

Reply:
i think a future pr would be a good idea. i'll change the platforms and broken status for this pr until we can get everything resolved